flexget/plugins/urlrewrite/newpct.py

```diff
@@ -24,14 +24,21 @@ class UrlRewriteNewPCT(object):
     def url_rewritable(self, task, entry):
         url = entry['url']
         rewritable_regex = '^http:\/\/(www.)?newpct1?.com\/.*'
-        return re.match(rewritable_regex, url) and not url.startswith('http://www.newpct.com/descargar/')
+        return re.match(rewritable_regex, url) and not url.endswith('.torrent')
 
     # urlrewriter API
     def url_rewrite(self, task, entry):
         entry['url'] = self.parse_download_page(entry['url'])
 
     @plugin.internet(log)
     def parse_download_page(self, url):
+        if 'newpct1' in url:
+            return self.parse_newpct1(url)
```

Comment:
I'm not a Python expert (I just read some high level points to be able to modify current code) so I'm not sure what to do exactly to reach your code quality expectations. (But I can try if you give some tips :))
All I know by now is that it works... and original newpct rewriter doesn't. So it may help others as it is now. ;-)